client/components/app.jsx

```diff
@@ -1,36 +1,37 @@
 import Axios from 'axios';
 import React from 'react';
-
+import DescriptionHalf from './DescriptionHalf.jsx'
+import CardComponent from './StyledComponents/CardComponent.js'
 class App extends React.Component {
     constructor(props) {
         super(props)
         this.state = {
-            initial: '',
-            data: ''
+            data: []
         };
     }
 
     componentDidMount() {
-        Axios.get('/getData')
+        Axios.get('/getData/2')
             .then((response) => {
                 this.setState({
-                    data: response.data
+                    data: response.data[0]
```

Comment:
Is this getting you one primary record? I was wondering if we were supposed to hard code to a given primary record, or just randomly choose one. Maybe a question I should ask Josh...

Reply:
The requested Location comes back from the server and is placed in an array. The location that comes back is a result of the the url => tripadvisor.com/1 through tripadvisor.com/100. 
Once the location comes back its placed in an array of 1 location, and thus selected using the 0th index.